pkg/solace/message/inbound_message.go

```diff
@@ -22,6 +22,10 @@ import (
 	"solace.dev/go/messaging/pkg/solace/message/rgmid"
 )
 
+// CacheRequestID - a type to be used for correlating received,
+// previously cached messages with their associated cache response.
+type CacheRequestID uint64
+
```

Comment:
is there value to this type?   internally we convert/cast  a C.solClient_uint64_t to a uint64 on return from the native api, then convert/cast that to a CacheRequestID?   Why don't we either cast directly to a CacheRequestID or just define as a uint64 in the interface?